bridge_ui/src/hooks/useGetSourceParsedTokenAccounts.ts

```diff
@@ -693,7 +689,34 @@ const getAlgorandParsedTokenAccounts = async (
       .accountInformation(walletAddress)
       .do();
     const parsedTokenAccounts: ParsedTokenAccount[] = [];
-    parsedTokenAccounts.push(
+    for (const asset of accountInfo.assets) {
+      const assetId = asset["asset-id"];
+      const amount = asset.amount;
+      const metadata = await fetchSingleMetadata(assetId, algodClient);
+      const isNFT: boolean = amount === 1 && metadata.decimals === 0;
```

Comment:
Won’t this exclude tokens of which the user owns exactly 1?

Reply:
metadata.decimals should cover that case.